toolbox/bulk_processing/validators.py

```diff
@@ -203,6 +203,15 @@ def validate(value, **_kwargs):
     return validate
 
 
+def alphanumeric_plus_hyphen_field_values_ignore_empty_strings():
+    def validate(value, **_kwargs):
+        stripped_field_value = value.replace("-", "")
+        if not stripped_field_value.isalnum() and stripped_field_value != "" or '':
```

Comment:
```suggestion
        if not stripped_field_value.isalnum() and stripped_field_value != "":
```
Should just be this, the `or ''` is equivalent to `or False`.